packages/cli/src/aws/index.ts

```diff
@@ -5,9 +5,16 @@ import { listStacksCommand } from './list';
 import { updateAppCommand } from './update-app';
 import { updateBucketPoliciesCommand } from './update-bucket-policies';
 import { updateServerCommand } from './update-server';
+import { createMedplumCommand } from '../util/command';
 
 export const aws = new Command('aws').description('Commands to manage AWS resources');
 
+const updateCmd = createMedplumCommand('update-server')
```

Comment:
Did this change anything?  If not, preference to leave it as-is to be consistent with the other commands in the file.